pkg/karmadactl/utils/idempotency.go

```diff
@@ -0,0 +1,87 @@
+package utils
```

Comment:
Do the functions of this file is same as `pkg/util/secret` etc, could we move it to `pkg/util`?

Reply:
Some functions about secret is similar with `pkg/util/secret` 
Do you have suggestion about how to move？
How about just move the file to `pkg/util` and delete the similar functions. Looks not unified
And the left function may be only used in karmada initial.

Reply:
> Or can we move the left to pkg/karmadactl/addons/utils

Agree.